SmoothConfig.Api/Controllers/ImporterController.cs

```diff
@@ -36,7 +43,18 @@ public IActionResult GenerateTokenizedConfig(IFormFile file)
             {
                 FileDownloadName = file.FileName
             };
+        }
+
+        [HttpPost]
+        [Authorize]
+        public IActionResult ImportConfig([FromForm] ImportConfigViewModel importConfigViewModel)
+        {
+            if (importConfigViewModel is null)
+                return UnprocessableEntity();
```

Comment:
We already discussed it, and we came to the conclusion that the idea would be a Bad Request